assets/js/feature-tours/unified-dashboard.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Unified Dashboard Feature Tour.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import { VIEW_CONTEXT_DASHBOARD } from '../googlesitekit/constants';
+import { isFeatureEnabled } from '../features';
+
+const unifiedDashboard = {
+	slug: 'unifiedDashboard',
+	contexts: [ VIEW_CONTEXT_DASHBOARD ],
+	version: '1.50.0',
+	gaEventCategory: ( viewContext ) => `${ viewContext }_unified-dashboard`,
+	checkRequirements: () => isFeatureEnabled( 'unifiedDashboard' ),
+	steps: [
+		{
+			target: '.googlesitekit-navigation',
+			title: __(
+				'New! Navigate your dashboard easily with the dashboard tabs',
+				'google-site-kit'
+			),
+			content: __(
+				'Jump straight to the relevant section in your dashboard with just one click - no more scrolling.',
+				'google-site-kit'
+			),
+			placement: 'bottom-start',
```

Comment:
I think if we use just `bottom`, it will put the tooltip in the center, won't it? If so, let's update this to be `bottom` because right now it looks weird:

![image](https://user-images.githubusercontent.com/873049/150852056-54850a3d-ba1d-41bb-8bc9-28fff5c617b6.png)

Reply:
Nice suggestion! Updated!